presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoSession.java

```diff
@@ -112,9 +112,9 @@ public MongoSession(TypeManager typeManager, MongoClient client, MongoClientConf
     {
         this.typeManager = requireNonNull(typeManager, "typeManager is null");
         this.client = requireNonNull(client, "client is null");
-        this.schemaCollection = config.getSchemaCollection();
+        this.schemaCollection = requireNonNull(config.getSchemaCollection(), "config.getSchemaCollection() is null");
         this.cursorBatchSize = config.getCursorBatchSize();
-        this.implicitPrefix = config.getImplicitRowFieldPrefix();
+        this.implicitPrefix = requireNonNull(config.getImplicitRowFieldPrefix(), "config.getImplicitRowFieldPrefix() is null");
```

Comment:
But you already validate that it is not non null.

Reply:
Yes, I know